AppKit/CPColor.j

```diff
@@ -467,6 +467,16 @@ var cachedBlackColor,
     return [[CPColor alloc] _initWithCSSString: aString];
 }
 
++ (CPColor)selectedTextBackgroundColor
+{
+    return [CPColor colorWithHexString:"99CCFF"];
+}
+
++ (CPColor)_selectedTextBackgroundColorUnfocussed
+{
+    return [CPColor colorWithHexString:"CCCCCC"];
```

Comment:
@Dogild I think that the selection color of CPTextField is not themed, isn't it?
what do you think of using the css system colors like this?
JavaScript ``` element.style.backgroundColor = 'Highlight';
element.style.color = 'HighlightText';
```